website/backend/impact/tests.py

```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from backend.impact.models import ImpactNumber
+
+class ImpactNumberTestCase(TestCase):
+    def setUp(self):
```

Comment:
Testing will require setting up a DB user to avoid the _"permission denied to create database"_  error